architecture/modules/functions/main.tf

```diff
@@ -89,6 +89,29 @@ resource "google_cloudfunctions_function" "process_dataset_file" {
   service_account_email = var.elpis_worker.email
 }
 
+resource "google_cloudfunctions_function" "delete_dataset_from_bucket" {
+  name        = "delete-dataset-from-bucket"
+  description = "Deletes a dataset from GCP when a dataset is deleted from Firestore"
+  runtime     = "python37"
+  region      = var.region
+
+  available_memory_mb   = 128
+  source_archive_bucket = google_storage_bucket.source.name
+  source_archive_object = google_storage_bucket_object.archive.name
+  entry_point           = "delete_dataset_from_bucket"
```

Comment:
You need to edit main.py to include this entrypoint, which I don't see in this PR so far.

Reply:
Thanks I just did that.